### ISSUE TITLE
[FORMATTING] postgresql timestamp datatype on create table adds unnecessary newline

### ISSUE THREAD

**Issue by not-rusty** [sql-formatter-org/sql-formatter#391]
**Input data**
Which SQL and options did you provide as input?

```sql
create table test (
	a int,
	b int,
	c TIMESTAMP with time zone,
	d int
);
```

**Expected Output**

```sql
CREATE    TABLE test (
                    a INT,
                    b INT,
                    c TIMESTAMP WITH TIME ZONE,
                    d INT
                    );
```

**Actual Output**

```sql
CREATE    TABLE test (
                    a INT,
                    b INT,
                    c TIMESTAMP
                    WITH      TIME ZONE,
                              d INT
                    );
```

**Usage**

- How are you calling / using the library? By coc-sql
- What SQL language(s) does this apply to? postgresql
- Which SQL Formatter version are you using? latest with coc-sql

**Comment by not-rusty** [comment]
The *with* keyword causes the issue. Trying with timestamp without time zone looks fine.

**Comment by nene** [comment]
This is similar to #340

- The formatter shouldn't attempt to detect a `WITH`-clause inside a `CREATE TABLE` statement.
- Like in #340 the formatter shouldn't detect `UPDATE` statement inside a `SELECT`.

No quick fix available for it, but we're moving towards a better architecture to solve this.

**Comment by nene** [comment]
Found a simpler way of fixing this. Released in 10.2.0.
